src/call.cpp

```diff
@@ -728,7 +727,25 @@ bool call::connect_socket_if_needed()
             ERROR_NO("Unable to bind UDP socket");
         }
     } else { /* TCP, SCTP or TLS. */
-        struct sockaddr_storage *L_dest = &remote_sockaddr;
+        /* Pick a random remote IP address from the list. Doing this here
+         * (just before opening a new connection) means that we'll round-robin our
```

Comment:
Doesn't round-robin mean something different to random, which is what we're actually doing?